.changeset/rotten-chefs-shake.md

```diff
@@ -0,0 +1,4 @@
+---
+---
+
+docs: add max outputs documentation to combining UTXO docs
```

Comment:
Should this be closing #3429, looks like it should just be related to?

Reply:
I put it as closing given we already have docs around combining UTXOs and this just seeks to add the last part of the issue, I clarified [here](https://github.com/FuelLabs/fuels-ts/issues/3429#issuecomment-2508797600)

Reply:
@maschad Since we already have docs related to combining UTXOs, perhaps we can now add docs related to dismantling UTXOs, which is splitting one into several.

Reply:
Yeah let's be as explicit as possible given this came up very recently.

Reply:
I've decided to add a function inspired by @luizstacio 's that splits the UTXOs for the consumers and documented it [here](https://github.com/FuelLabs/fuels-ts/pull/3435/files#diff-27051d152c0be799da72bb993355ee169e2b7a9457d003fdba0c1573e16f9951)